homeassistant/components/prometheus/__init__.py

```diff
@@ -324,12 +324,14 @@ def _handle_climate(self, state):
                 )
 
     def _handle_humidifier(self, state):
-        humidity = state.attributes.get(ATTR_HUMIDITY)
-        if humidity:
+        humidifer_target_humidity_percent = state.attributes.get(ATTR_HUMIDITY)
+        if humidifer_target_humidity_percent:
             metric = self._metric(
-                "humidity", self.prometheus_cli.Gauge, "Target Relative Humidity"
+                "humidifer_target_humidity_percent",
+                self.prometheus_cli.Gauge,
+                "Target Relative Humidity",
             )
-            metric.labels(**self._labels(state)).set(humidity)
+            metric.labels(**self._labels(state)).set(humidifer_target_humidity_percent)
```

Comment:
I think you have a typo here: it should be `humidifier` instead of `humidifer`.

Reply:
Sorry, I've just realized it's actually my typo from a code review comment :) In any case I'd appreciate if you could fix it.

Reply:
Yeap, copied from your comment ;)
Fixed. Thanks for the review!